nvflare/private/fed/app/client/worker_process.py

```diff
@@ -74,61 +76,55 @@ def main():
     config_folder = kv_list.get("config_folder", "")
     secure_train = kv_list.get("secure_train", True)
     if config_folder == "":
-        args.client_config = "config_fed_client.json"
+        args.client_config = WorkspaceConstants.CLIENT_JOB_CONFIG
     else:
-        args.client_config = os.path.join(config_folder, "config_fed_client.json")
+        args.client_config = os.path.join(config_folder, WorkspaceConstants.CLIENT_JOB_CONFIG)
     args.config_folder = config_folder
     args.env = os.path.join("config", "environment.json")
+    workspace = Workspace(args.workspace, args.client_name, config_folder)
 
     try:
-        remove_restart_file(args)
+        remove_restart_file(workspace)
     except BaseException:
         print("Could not remove the restart.fl / shutdown.fl file.  Please check your system before starting FL.")
         sys.exit(-1)
 
-    restart_file = os.path.join(args.workspace, "restart.fl")
+    restart_file = workspace.get_file_path_in_root("restart.fl")
     if os.path.exists(restart_file):
         os.remove(restart_file)
 
-    print("starting the client .....")
+    # Initialize audit service since the job execution will need it!
+    audit_file_name = workspace.get_audit_file_path()
+    AuditService.initialize(audit_file_name)
 
-    startup = os.path.join(args.workspace, "startup")
-    SecurityContentService.initialize(content_folder=startup)
+    # print("starting the client .....")
+
+    SecurityContentService.initialize(content_folder=workspace.get_startup_kit_dir())
 
     thread = None
     stop_event = threading.Event()
     deployer = None
     client_app_runner = None
     federated_client = None
 
-    startup = args.startup
-    app_root = os.path.join(
-        args.workspace,
-        WorkspaceConstants.WORKSPACE_PREFIX + str(args.job_id),
-        WorkspaceConstants.APP_PREFIX + args.client_name,
-    )
-
-    logging_setup(app_root, args, config_folder, startup)
-
-    log_file = os.path.join(args.workspace, args.job_id, "log.txt")
-    add_logfile_handler(log_file)
-    logger = logging.getLogger("worker_process")
-    logger.info("Worker_process started.")
+    app_root = workspace.get_app_dir(str(args.job_id))
 
     try:
         # start parent process checking thread
         thread = threading.Thread(target=check_parent_alive, args=(parent_pid, stop_event))
         thread.start()
 
         conf = FLClientStarterConfiger(
-            app_root=startup,
-            client_config_file_name=args.fed_client,
-            log_config_file_name=args.log_config,
+            workspace=workspace,
             kv_list=args.set,
-            logging_config=False,
         )
         conf.configure()
 
+        log_file = workspace.get_app_log_file_path(args.job_id)
+        add_logfile_handler(log_file)
+        logger = logging.getLogger("worker_process")
+        logger.info("Worker_process started.")
+
```

Comment:
should we call the function `configure_logging` here?

Reply:
Will address this in another PR